specs/nonbibrecord.proto

```diff
@@ -25,6 +25,7 @@ message NonBibRecord {
     repeated string bibgroup = 23;
     repeated string bibgroup_facet = 24;
     repeated string gpn = 26;
+    repeated string uat = 28;
```

Comment:
I should have caught this with the gpn field, but why did we skip tags 25 and 27?

Reply:
I had used 25 and 27 once and rebuilt the protobufs, but then realised that I had messed something up. the documentation suggested that once a tag is assigned and built, it is not recommended to edit it because it can have weird and bad effects. so just to be on the safe side I skipped them